app/views/settings/index.html.erb

```diff
@@ -18,6 +18,8 @@
           <div class="actions">
             <%= f.submit "Update", class: "btn-lg btn-primary btn-block" %>
           </div>
+          <h4 class="text-center"></h4>
+          <%= link_to "allow your account to be signed in with twitter", 'settings/auth/twitter', class: 'btn-lg btn-primary btn-block text-center' %>
```

Comment:
In order for the oauth gem to do its thing, there *HAS* to be a call to `/auth/twitter/`